src/common/layout/ContentPage.js

```diff
@@ -7,7 +7,7 @@ import { Switch, Route } from 'react-router-dom';
 import { Desktop } from '../Responsive';
 import Header from './Header';
 import StaticFAQ from '../../StaticFAQ';
-import FAQArticleDetail from '../../StaticFAQ/FAQArticleDetail';
+import FAQArticleDetail from '../../StaticFAQ/ArticleDetail/FAQArticleDetail';
```

Comment:
`ArticleDetail/FAQArticleDetail` sounds a bit too redundant. Also we have `FAQ` twice in the path.

Reply:
So maybe renaming `FAQArticleDetail` would remove both redundancies

Reply:
fixed